proj4/src/main/scala/geotrellis/proj4/io/wkt/WKTParser.scala

```diff
@@ -0,0 +1,152 @@
+    case _ ~ name ~ _ ~ datumType ~ _ ~ authority ~ _ => new VertDatum(name, datumType, authority)
+  }
+
+  def geogcs: Parser[Geogcs] = """GEOGCS[""" ~ string ~ comma ~ datum ~ comma ~ primeM ~ comma ~ unit ~ comma.? ~ getAxisList.? ~ comma.? ~ authority.? ~ """]""" map {
+    case _ ~ name ~ _ ~ datum ~ _ ~ primeM ~_ ~ unit ~ _~ axes ~ _ ~ auth ~ _ => new Geogcs(name, datum, primeM, unit, axes, auth)
+  }
+
+  def projection: Parser[Projection] = """PROJECTION[""" ~ string ~ comma.? ~ authority.? ~ """]""" map {
+    case _  ~ name ~ _ ~ authority ~ _ => new Projection(name, authority)
+  }
+
+  def parameterList: Parser[List[Parameter]] = ((parameter ~ comma.?)*) ^^ (_.map(_._1))
+
+  def projcs: Parser[Projcs] = """PROJCS[""" ~ string ~ comma ~ geogcs ~ comma ~ projection ~ comma.? ~ parameterList.? ~ comma.? ~ unit ~ comma.? ~ twinAxis.? ~ comma.? ~ authority.? ~ """]""" map {
+    case _  ~ name ~  _ ~ geogcs ~ _ ~ projection ~ _ ~ params ~ _  ~ unit ~ _ ~ twin ~ _ ~ authority ~ _ => new Projcs(name, geogcs, projection, params, unit, twin, authority)
+  }
+
+  def vertcs: Parser[VertCS] = """VERT_CS[""" ~ string ~ comma ~ vertDatum ~ comma ~ unit ~ comma.? ~ axis.? ~ comma.? ~ authority.? ~ """]""" map {
+    case _ ~ name ~ _ ~ vertDatum ~ _ ~ unit ~ _ ~ axis ~ _ ~ authority ~ _ => new VertCS(name, vertDatum, unit, axis, authority)
+  }
+
+  def localDatum: Parser[LocalDatum] = """LOCAL_DATUM[""" ~ string ~ comma ~ int ~ comma.? ~ authority.? ~ """]""" map {
+    case _ ~ name ~ _ ~ datumType ~ _ ~ authority ~ _ => new LocalDatum(name, datumType, authority)
+  }
+
+  def getAxisList: Parser[List[Axis]] = ((axis ~ comma.?)+)^^ (_.map(_._1))
+
+  def localCS: Parser[LocalCS] = """LOCAL_CS[""" ~ string ~ comma ~ localDatum ~ comma ~ unit ~ comma ~ getAxisList ~ comma.? ~ authority.? ~ """]""" map {
+    case _ ~ name ~ _ ~ localDatum ~_ ~ unit ~ _ ~ axisList ~ _ ~ authority ~ _ => new LocalCS(name, localDatum, unit, axisList, authority)
+  }
+
+  def geoccs: Parser[Geoccs] = """GEOCCS[""" ~ string ~ comma ~ datum ~ comma ~ primeM ~ comma ~ unit ~ comma.? ~ getAxisList.? ~ comma.? ~ authority.? ~ """]""" map {
+    case _ ~ name ~ _ ~ datum ~ _ ~ primeM ~ _ ~ unit ~ _ ~ getAxisList ~ _ ~ auth ~ _ => new Geoccs(name, datum, primeM, unit, getAxisList, auth)
+  }
+
+  def coordinateSys: Parser[Any] = geogcs | geoccs | projcs | vertcs | compdcs | localCS
+
+  def compdcs: Parser[CompDCS] = """COMPD_CS[""" ~ string ~ comma ~ coordinateSys ~ comma ~ coordinateSys ~ comma.? ~ authority.? ~ """]""" map {
+    case _ ~ name ~ _ ~ head ~ _ ~ tail ~ _ ~ auth ~ _ => new CompDCS(name, head, tail, auth)
+  }
+
+  def parseWKT: Parser[Tree] = localCS | projcs | geogcs | geoccs | compdcs | vertcs
+
+  def createTree() = {
+    //read input from epsg.properties file
+    var count = 0;
+    withWktFile { lines =>
+      for (line <- lines) {
+        //split the line for parsing the wkt
+        val array = line.split("=")
```

Comment:
As we found WKT text may contain `=` character as well, which makes this use of `split` dangerous. Need to be a little more careful and use `String. indexOf` and `String.subSequence` here.